doc/backend_vars.asciidoc

```diff
@@ -8,7 +8,7 @@ Supported variables per backend
 Variable;Values allowed;Default value;Explanation
 INCLUDE_MODULES;string;;comma separated names or fullnames of test modules to be included while excluding all that do not match, e.g. "boot,mod1"
 EXCLUDE_MODULES;string;;comma separated names or fullnames of test modules to exclude. Can be combined with INCLUDE_MODULES and has precedence, e.g. to additionally exclude modules based on an include-list
-SCHEDULE;string;;comma separated list of relative paths to test modules within CASEDIR without the implicit file extension '.pm' to be scheduled instead of evaluating a schedule from the test distributions main.pm file, e.g. "boot,console/mod1"
+SCHEDULE;string;;comma separated list of relative paths to test modules within *CASEDIR* without the implicit file extension '.pm' to be scheduled overriding the distribution's main.pm file. _e.g. for https://github.com/os-autoinst/os-autoinst-distri-opensuse: "SCHEDULE=tests/boot/boot_to_desktop,tests/console/mod1"_.
```

Comment:
sorry but "overriding" still sounds like that the file would be ignored. What is wrong with my previous way of phrasing?

Reply:
yes, works for me :)